css/syntaxes.json

```diff
@@ -172,11 +174,12 @@
   "single-animation-fill-mode": "none | forwards | backwards | both",
   "single-animation-iteration-count": "infinite | <number>",
   "single-animation-play-state": "running | paused",
-  "single-timing-function": "<single-transition-timing-function>",
+  "single-timing-function": "linear | <cubic-bezier-timing-function> | <step-timing-function> | <frames-timing-function>",
   "single-transition": "[ none | <single-transition-property> ] || <time> || <single-transition-timing-function> || <time>",
-  "single-transition-timing-function": "ease | linear | ease-in | ease-out | ease-in-out | step-start | step-end | steps( <integer> [, [ start | end ] ]? ) | cubic-bezier( <number>, <number>, <number>, <number> )",
+  "single-transition-timing-function": "ease | linear | ease-in | ease-out | ease-in-out | step-start | step-end | frames( <integer> ) | steps( <integer> [, [ start | end ] ]? ) | cubic-bezier( <number>, <number>, <number>, <number> )",
```

Comment:
This should be `single-transition-timing-function": "<single-timing-function>"` now that we defined the other syntaxes, right?